functions/package.json

```diff
@@ -13,29 +13,38 @@
     "test:watch": "jest --watch"
   },
   "engines": {
-    "node": "10"
+    "node": "12"
```

Comment:
Please change this to version 10 since this the current version supported by firebase